include/searchresultsformaction.h

```diff
@@ -0,0 +1,43 @@
+#ifndef NEWSBOAT_SEARCHRESULTSFORMACTION_H_
+#define NEWSBOAT_SEARCHRESULTSFORMACTION_H_
+#include "configcontainer.h"
+#include "itemlistformaction.h"
+#include "logger.h"
+#include "regexmanager.h"
+#include "rssfeed.h"
+#include "view.h"
+#include "keymap.h"
+#include <stack>
+
+namespace newsboat {
+
+class SearchResultsFormAction : public ItemListFormAction {
```

Comment:
It's a bit confusing that the class is named "SearchResults…" when other constants say "search results **list**". Can this be renamed to `SearchResultsListFormAction`? (Don't forget to rename the files and the include guards too!)

It's a long name, but at least it's consistent with all the other stuff we already have.

Reply:
https://github.com/newsboat/newsboat/pull/2031/commits/de6f4dce56fa5e8e9b21f22013d8fee4342af815